recipe/meta.yaml

```diff
@@ -24,14 +24,13 @@ package:
 
 source:
   url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cccl/{{ platform }}/cuda_cccl-{{ platform }}-{{ version }}-archive.{{ extension }}
-  sha256: e1636f27a142d24e73dfd831c54bbf5575b498fd5900648d7372fae46f824fdf  # [linux64]
-  sha256: 3304e563ed089be1129f79d8e45c9badc8eeba155c8b672f9659f223494edcd9  # [ppc64le]
-  sha256: 171073fd6557360b9db7f8559e17b1bb55679aadd5158681318ed9be67e54667  # [aarch64]
-  sha256: 908742d8f3c6fdd6d1d6316a7b919b5c506474f9551f491aa7335cb4f50bffbd  # [win]
+  sha256: 837c4e27939478ffa3d2ad18f9a109b83c05afbe891769732c5c615e53081fc1  # [linux64]
+  sha256: 7052855e3624aeb4abf91b0b9deed68f91054f898b85abef1d511c29ce544cc2  # [aarch64]
+  sha256: 748c07d466556b1562cc46a0389948af1e3fd92cc667ebf697bf6a9a30e736de  # [win]
 
 build:
-  number: 2
-  skip: true  # [osx]
+  number: 0
+  skip: true  # [osx or ppc64le]
```

Comment:
Just a reminder that when we add this `skip`, we also need to re-render

Reply:
Was planning on re-rendering once packages are available. Still not catching `cccl` with `conda search` despite anaconda web ui showing a download (noticed this on a few others today, but nothing drastic)

Reply:
Sounds good. Figured that might be the case. Just wanted to mention if not

Ok did see some non-zero download counts. So thought it was worth giving it a try

Since this is mirroring to CDN, it can be the case that different edges are updated earlier than others. Meaning availability could vary

Reply:
Looks like you had better timing and we're all green here now, thanks John!